gc/base/standard/ConfigurationGenerational.hpp

```diff
@@ -54,6 +54,16 @@ class MM_ConfigurationGenerational : public MM_ConfigurationStandard
 
 	virtual void defaultMemorySpaceAllocated(MM_GCExtensionsBase *extensions, void* defaultMemorySpace);
 	
+
+#if defined(J9VM_OPT_CRIU_SUPPORT)
+	/**
+	 * Startup gc threads on restore
+	 *
+	 * @param env[in] - the current environment
+	 */
```

Comment:
Use the correct formatting for the function description: https://github.com/eclipse/omr/blob/master/doc/CodingStandard.md#comment-function-parameters-clearly
```suggestion
	/**
	 * Startup GC threads on restore
	 *
	 * @param[in] env the current environment
	 * @return void
	 */
```